README.md

```diff
@@ -1,41 +1,7 @@
 # IoTtalk2.0 Javascript SDK
 
-Provide DAN of IoTtalk2.0 and some examples.  
-
-## Docuement
-
-_dan2.js_ depend on _jQuery_ and _Paho.MQTT_. You can add below three line in your header in order to use it.
-```html
-<script src="https://iottalk.github.io/iottalk-js/jquery.js"></script>
-<script src="https://iottalk.github.io/iottalk-js/mqttws31.min.js"></script>
-<script src="https://iottalk.github.io/iottalk-js/dan2.js"></script>
+## Development
+```
+npm install
+webpack
 ```
-
-### dan2.register(url, params, init_callback)
```

Comment:
What is the perpose of removing those API documentation?
Do those API be changed?